crates/katana/node/src/lib.rs

```diff
@@ -290,7 +291,9 @@ pub async fn spawn<EF: ExecutorFactory>(
 
     let middleware = tower::ServiceBuilder::new()
         .option_layer(cors)
-        .layer(ProxyGetRequestLayer::new("/", "health")?)
+        .layer(DevnetProxyLayer::new("/", "health")?)
+        .layer(DevnetProxyLayer::new("/account_balance", "dev_accountBalance")?)
+        .layer(DevnetProxyLayer::new("/fee_token", "dev_feeToken")?)
```

Comment:
we don't need to modify the `/health` endpoint, we can keep using the `ProxyGetRequestLayer` for it.